src/api.rs

```diff
@@ -133,6 +142,94 @@ pub(crate) fn deal_cards(cookies: &CookieJar<'_>) -> Json<Value> {
     Json(json!({"spread": spread, "message": "New Spread preserved"}))
 }
 
+#[derive(Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub(crate) struct SnackPurchase {
+    pub(crate) snack_id: String,
+}
+
+#[post("/api/buySnackNoUpgrade", data = "<purchase>")]
+pub(crate) fn buy_snack(
+    cookies: &CookieJar<'_>,
+    purchase: Json<SnackPurchase>,
+) -> (Status, Json<Value>) {
+    if purchase.snack_id == "Peanuts" {
+        let peanuts = cookies
+            .get_pending("peanuts")
+            .and_then(|t| t.value().parse::<i32>().ok())
+            .unwrap_or(0)
+            + 1000;
+
+        cookies.add(crate::new_cookie("peanuts", peanuts.to_string()));
+        (
+            Status::Ok,
+            Json(json!({
+                "message": format!("You receive 1000 peanuts. You now have {} peanuts", peanuts)
+                // not sure this message is accurate for expansion era?
+            })),
+        )
+    } else {
+        let message = choose(ERROR_MESSAGES);
+        (
+            Status::BadRequest,
+            Json(json!({
+                "error": message,
+                "message": message,
+            })),
+        )
+    }
+}
+
+#[post("/api/buyADangSquirrel")]
+pub(crate) fn buy_a_dang_squirrel(cookies: &CookieJar<'_>) -> Json<Value> {
+    cookies.add(crate::new_cookie(
+        "squirrels",
+        (cookies
+            .get_pending("squirrels")
+            .and_then(|t| t.value().parse::<i32>().ok())
+            .unwrap_or(0)
+            + 1)
+        .to_string(),
+    ));
+    Json(json!({"message": "Bought a squirrel."}))
+}
+
+#[derive(Deserialize)]
+pub(crate) struct EatADangPeanut {
+    pub(crate) amount: i32,
+}
+
+#[post("/api/eatADangPeanut", data = "<dang_peanut>")]
+pub(crate) fn eat_a_dang_peanut(
+    cookies: &CookieJar<'_>,
+    dang_peanut: Json<EatADangPeanut>,
+) -> Json<Value> {
+    cookies.add(crate::new_cookie(
+        "peanuts",
+        (cookies
+            .get_pending("peanuts")
+            .and_then(|t| t.value().parse::<i32>().ok())
+            .unwrap_or(0)
+            - dang_peanut.amount)
+            .to_string(),
+    ));
+    Json(json!({}))
+}
+
+#[post("/api/buyADangPeanut")]
+pub(crate) fn buy_a_dang_peanut(cookies: &CookieJar<'_>) -> Json<Value> {
+    let peanuts = cookies
+        .get_pending("peanuts")
+        .and_then(|t| t.value().parse::<i32>().ok())
+        .unwrap_or(0)
+        + 1000;
```

Comment:
haha um

so this actually relies on the value of temporal.doc.alpha which i set to a random number prior to when it was fixed at 1000 https://github.com/iliana/before/blob/03962c260c7b987bab0f55f7da4c5856e542d26b/src/events.rs#L844

so we might want to make this less random so the two systems can be in sync? or maybe when we randomly generate a new alpha we set a cookie with the value?